### DISCUSSION TITLE
I'm new to GitHub and I want to make a repository to fetch my website

### DISCUSSION THREAD

**Discussion by Github2503** [community/community#23566] [New to GitHub]
<p>Ok, I’m new to GitHub (and don’t have any kind of coding experience) so I want to start by doing something simple: I want to create a GitHub repo to simply fetch my website (<a href="https://aboutrock2.godaddysites.com/" rel="noopener nofollow ugc">https://aboutrock2.godaddysites.com/</a>)! Afterward, I will integrate “Cloudflare Pages” into the GitHub repo (<span class="spoiler">so I can run it through the Cloudflare network</span>)! But first, let’s focus on getting the website fetched! I’ve already created a GitHub repository (but haven’t added any files to it yet!) The repo is embedded below (in case we need it later)</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/Github2503/AboutRock2">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/Github2503/AboutRock2" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/96203580/181077933-cfb3a656-a1f5-4f3a-b4e3-b25859235843.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/2/72e36656440b8d6c3f31d790b398959ca9b45a2c_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/2/72e36656440b8d6c3f31d790b398959ca9b45a2c_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/2/72e36656440b8d6c3f31d790b398959ca9b45a2c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/2/72e36656440b8d6c3f31d790b398959ca9b45a2c_2_10x10.png"></div>

<h3><a href="https://github.com/Github2503/AboutRock2" target="_blank" rel="noopener nofollow ugc">Github2503/AboutRock2</a></h3>

  <p>Contribute to Github2503/AboutRock2 development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I’m looking forward to using GitHub!</p>

**Comment by jfullstackdev** [comment]
<p>I’m not very sure of your GoDaddy Website Builder, but maybe this is the one you are looking for,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://blog.devgenius.io/setting-up-git-with-github-and-a-godaddy-vps-virtual-private-server-f74f7cf64429">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181077934-2a41071e-948d-4e7b-8de2-45e9bdfcd645.jpeg" class="site-icon" width="256" height="256">

      <a href="https://blog.devgenius.io/setting-up-git-with-github-and-a-godaddy-vps-virtual-private-server-f74f7cf64429" target="_blank" rel="noopener" title="01:30PM - 14 June 2020">Medium – 14 Jun 20</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/457;"><img src="https://user-images.githubusercontent.com/47092464/181077935-e54ed3a9-3eb6-4b64-a717-28a1b2f55ba1.jpeg" class="thumbnail" width="690" height="457" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/b/6b8c3be842f2f0435fb88dcc152a6ae738d241e9_2_690x457.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/6/b/6b8c3be842f2f0435fb88dcc152a6ae738d241e9_2_1035x685.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/b/6b8c3be842f2f0435fb88dcc152a6ae738d241e9.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/b/6b8c3be842f2f0435fb88dcc152a6ae738d241e9_2_10x10.png"></div>

<h3><a href="https://blog.devgenius.io/setting-up-git-with-github-and-a-godaddy-vps-virtual-private-server-f74f7cf64429" target="_blank" rel="noopener">Setting Up Git with Github and a Godaddy VPS (Virtual Private Server)</a></h3>

  <p>A quick guide to get you set up and running with git between your dev machine and a VPS.</p>

  <p>
    <span class="label1">Reading time: 6 min read</span>
  </p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by Github2503** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/8/78269bc14649e4d51fb036ad7618a303803d8b90.png" data-download-href="/uploads/short-url/h8U0U9qv16IimrUaLwqqXz8JSKs.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/96203580/181077936-4be76ee2-7449-4259-bddf-267916e46a05.png" alt="image" data-base62-sha1="h8U0U9qv16IimrUaLwqqXz8JSKs" width="690" height="167" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/8/78269bc14649e4d51fb036ad7618a303803d8b90_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">694×168 13.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>It mentions Linux, and I have Chrome OS so I’m not sure if it will work!</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="Github2503" data-post="3" data-topic="223040">
<div class="title">
<div class="quote-controls"></div>
 Github2503:</div>
<blockquote>
<p>I have Chrome OS</p>
</blockquote>
</aside>
<p>check this out,</p>
<p><a href="https://www.techrepublic.com/article/how-to-use-github-in-chrome-os/#:~:text=In%20order%20to%20get%20your,you're%20ready%20to%20continue.">https://www.techrepublic.com/article/how-to-use-github-in-chrome-os/#:~:text=In%20order%20to%20get%20your,you’re%20ready%20to%20continue.</a></p>

**Comment by Github2503** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/f/3fb60184754f07a5cdb915c9d145cb34c3a625fb.png" data-download-href="/uploads/short-url/95C1E14boCBk6TbIjoILmE6TrN9.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/96203580/181077941-9904a0e1-facc-40fd-bea6-b6125d3495e4.png" alt="image" data-base62-sha1="95C1E14boCBk6TbIjoILmE6TrN9" width="420" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3fb60184754f07a5cdb915c9d145cb34c3a625fb_2_420x500.png, https://aws1.discourse-cdn.com/github/original/3X/3/f/3fb60184754f07a5cdb915c9d145cb34c3a625fb.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/f/3fb60184754f07a5cdb915c9d145cb34c3a625fb.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3fb60184754f07a5cdb915c9d145cb34c3a625fb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">484×575 27.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Well, I’m screwed, aren’t I?</p>
<p>Is there any other way I could complete the repo? <strong>Remember: I use ChromeOS</strong></p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="Github2503" data-post="5" data-topic="223040">
<div class="title">
<div class="quote-controls"></div>
 Github2503:</div>
<blockquote>
<p>Remember: I use ChromeOS</p>
</blockquote>
</aside>
<p>check this out,</p>
<p><a href="https://nosarthur.github.io/coding/2018/01/15/termux.html">https://nosarthur.github.io/coding/2018/01/15/termux.html</a></p>

**Comment by Github2503** [comment]
<p><img src="https://user-images.githubusercontent.com/96203580/181077943-d2a5bbeb-5bfd-4547-af5f-97778da6d6aa.png" alt="image" data-base62-sha1="egi2fIJk1DtkaNDcS9vlQ8NSNHX" width="262" height="18"></p>
<p>Administrative policies on my Chromebook (yeah, it’s enterprise managed) <strong>prevent</strong> DEV mode from being enabled!</p>
<p>I don’t know if that means I’m fucked or what, but do you have any other fixes, maybe?</p>

**Comment by jfullstackdev** [comment]
<p>If I’m getting your situation correctly, this is the answer from Google Support forum,</p>
<blockquote>
<p>If you have a Chromebook that has been issued by a school or company, you cannot remove the management. There are no hacks or workarounds. It cannot be done.</p>
<p>If you have a Chromebook that you bought, but you signed on with a managed account, then management has taken over your Chromebook. You will need to <a href="https://support.google.com/chromebook/answer/183084?hl=en&amp;source=genius-helpouts">Reset your Chromebook - Chromebook Help</a> and then never sign in with a managed account.</p>
<p>If you bought a used Chromebook and found that it’s managed, then return the Chromebook. It is the responsibility of the school or company that managed the Chromebook to unenroll the Chromebook from management BEFORE selling it.</p>
</blockquote>
<p><a href="https://support.google.com/chromebook/thread/4647238/how-do-i-take-off-the-manage-part-off-my-computer?hl=en">https://support.google.com/chromebook/thread/4647238/how-do-i-take-off-the-manage-part-off-my-computer?hl=en</a></p>

**Comment by Github2503** [comment]
<p>So I <strong>can’t</strong> do it (unless they remove the management)?</p>

**Comment by jfullstackdev** [comment]
<p>which situation is yours?</p>
<p>if you know who to contact to remove it as managed by, say, the IT of the school or a company or a managed account,</p>
<p>but if that is an issued Chromebook, well, you can still request them but there is no assurance they will grant your request,</p>

**Comment by Github2503** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="10" data-topic="223040">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>but if that is an issued Chromebook, well, you can still request them but there is no assurance they will grant your request,</p>
</blockquote>
</aside>
<p>Yeah, but it’s probably <strong>very</strong> unlikely they will grant the request!</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>I suggest if that is an issued one by your company or school, and there is no solution, to buy your own, I used a very low-end laptop for 4 years, where I had learned everything, it was just an HP Celeron processor, and was very cheap,</p>
<p>you can pick, say an i3, which is not so bad compared to Celeron ( and still affordable ), and continue making progress, so you are free to install everything,</p>

**Comment by Github2503** [comment]
<p>I guess that would work!</p>
